vscode-dotnet-runtime-library/src/test/unit/TestUtility.ts

```diff
@@ -58,29 +59,79 @@ export function getMockAcquisitionWorkerContext(acquireContext : IDotnetAcquireC
     return workerContext;
 }
 
-export function getMockAcquisitionWorker(mockContext : IAcquisitionWorkerContext) : MockDotnetCoreAcquisitionWorker
+export function getMockAcquisitionWorker(mockContext: IAcquisitionWorkerContext): MockDotnetCoreAcquisitionWorker
 {
     const acquisitionWorker = new MockDotnetCoreAcquisitionWorker(getMockUtilityContext(), new MockVSCodeExtensionContext());
     return acquisitionWorker;
 }
 
-export function getMockUtilityContext() : IUtilityContext
+export function getMockUtilityContext(): IUtilityContext
 {
-    const utilityContext : IUtilityContext = {
-        ui : new MockWindowDisplayWorker(),
-        vsCodeEnv : new MockVSCodeEnvironment()
+    const utilityContext: IUtilityContext = {
+        ui: new MockWindowDisplayWorker(),
+        vsCodeEnv: new MockVSCodeEnvironment()
     }
     return utilityContext;
 }
 
-export function getMockAcquireContext(nextAcquiringVersion : string, arch : string | null | undefined, installMode : DotnetInstallMode) : IDotnetAcquireContext
+export function getMockAcquireContext(nextAcquiringVersion: string, arch: string | null | undefined, installMode: DotnetInstallMode): IDotnetAcquireContext
 {
-    const acquireContext : IDotnetAcquireContext =
+    const acquireContext: IDotnetAcquireContext =
     {
         version: nextAcquiringVersion,
         architecture: arch,
         requestingExtensionId: 'test',
         mode: installMode
     };
     return acquireContext;
+}
+
+export async function getDistroInfo(context: IAcquisitionWorkerContext): Promise<DistroVersionPair>
+{
+    return new LinuxVersionResolver(context, getMockUtilityContext()).getRunningDistro();
+}
+
+/**
+ *
+ * @param distroInfo The distro and version of the system
+ * @returns The built-in distro supported version of the .NET SDK.
+ * Only maintaining the microsoft supported versions for now.
+ */
+export async function getLinuxSupportedDotnetSDKVersion(context: IAcquisitionWorkerContext, distroInfo?: DistroVersionPair): Promise<string>
+{
+    distroInfo ??= await getDistroInfo(context);
+
+    if (distroInfo.distro === UBUNTU_DISTRO_INFO_KEY)
+    {
+        if (distroInfo.version < '22.04')
+        {
+            return '6.0.100';
+        }
+        if (distroInfo.version < '24.04')
+        {
+            return '9.0.100';
+

```

Comment:
nit: extra newline

Reply:
Ah, good catch, this got added in the above commit.